src/main/java/algebra/fft/README.md

```diff
@@ -38,4 +38,4 @@ JavaPairRDD<Long, FieldT> radix2CosetInverseFFT(final JavaPairRDD<Long, FieldT>
 [CLRS] [_Introduction to Algorithms_](https://mitpress.mit.edu/books/introduction-algorithms), Thomas H. Cormen, Charles E. Leiserson, Ronald L. Rivest, and Clifford Stein, 2009
 
 [S11] [_Schönhage-Strassen Algorithm with MapReduce
-for Multiplying Terabit Integers_](https://people.apache.org/~szetszwo/ssmr20110429.pdf), Tsz-Wo Sze, 2011
\ No newline at end of file
+for Multiplying Terabit Integers_](https://people.apache.org/~szetszwo/ssmr20110429.pdf), Tsz-Wo Sze, 2011
```

Comment:
Is this change necessary?

Reply:
No, I didn't change anything here.

Reply:
Can I create new PR and change it?

Reply:
Yes

Reply:
I’m not sure how to fix it, sorry. I created a new PR and there is the same error.